android-project/app/src/main/java/oceantreasur/es/android_project/BaseAnswerActivity.java

```diff
@@ -0,0 +1,91 @@
+package oceantreasur.es.android_project;
+
+import android.content.Intent;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+
+import com.bumptech.glide.Glide;
+
+public abstract class BaseAnswerActivity extends AppCompatActivity {
+
+    private ImageView image;
+    private oceantreasur.es.android_project.CustomTextView answerMessage;
+    private oceantreasur.es.android_project.CustomTextView answerWord;
+    private ProgressBar progressBar;
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_answer);
+
+        Bundle intentData = getIntent().getExtras();
+
+        final String imageUrl = intentData.getString("URL");
+        final String word =     intentData.getString("WORD");
```

Comment:
Check if the extras are present before assigning them to variables, using Intent.hasExtra(String name)  and throw IllegalArgumentException if they are not (making the extras mandatory by this Activity).